backend_tsc/src/entitites/user.ts

```diff
@@ -0,0 +1,24 @@
+import {Schema,model,Model} from "mongoose"
+import {IMongo_Entity} from "./mongo_entity";
+
+export interface IUser extends IMongo_Entity{
+    username:string,
+    password:string,
+    active:boolean
+    role:string
+
+}
+
+type UserModel = Model<IUser>;
+const userSchema= new Schema<IUser,UserModel>({
+    username:{type:String, required:true,trim:true},
+    password:{type:String, required:true,trim:true},
+    active:{type:Boolean, required:true, default:true},
+    role:{type:String,required:true,enum:["admin","tester"]}//wtf is tester?
```

Comment:
What happens when we remove the tester role? Does anything break?

Reply:
Can confirm we do not need it, not until we start implementing roles beyond admin/not-admin. Even having it for testing doesn't really make sense. If we want to test the backend, we should be using mocks like `jest` to fake DB responses and such. Having a "tester" user with DB access could be a security risk.